.github/scripts/check-engines-cli.sh

```diff
@@ -52,36 +52,32 @@ esac
 echo "Assumed OS: $os_name"
 echo "CLI_QUERY_ENGINE_TYPE == $CLI_QUERY_ENGINE_TYPE"
 
+ENGINES_PACKAGE=$(node -e "console.log(path.dirname(require.resolve('@prisma/engines/package.json', {paths: [path.dirname(require.resolve('prisma/package.json'))]})))")
```

Comment:
This looks much nicer than before! Can we also echo it, just in case for the future?